### DISCUSSION TITLE
How can I get administrator rights now?

### DISCUSSION THREAD

**Discussion by viewfirst** [Pryaxis/TShock#2141] [Q&A]
My English is very poor, the following is from Google Translate

I used docker to set up a private server, but I didn't record the 8 digits behind /setup. How can I get administrator rights now?

**Comment by hakusaro** [comment]
If you didn't see it, try restarting the Docker container and looking at

the logs. The code is also stored in a file called `authcode.txt` in the

server folder.

On October 24, 2020, GitHub <notifications@github.com> wrote:
> My English is very poor, the following is from Google Translate
>
> I used docker to set up a private server, but I didn't record the 8
> digits behind /setup. How can I get administrator rights now?
>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/2141>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAECBJ4WRIHHTHNGFCXOY5LSMMDMHANCNFSM4S5W273Q>.

**Reply by viewfirst** [reply]
For some reason, I have deleted the docker container and selected an existing world when creating a new server. The file you mentioned is not in the "world" folder. Is there any other way to help me?

**Reply by hakusaro** [reply]
Which container are you using? Can you link to the dockerhub entry?

**Reply by viewfirst** [reply]
https://hub.docker.com/r/ryshe/terraria

**Reply by viewfirst** [reply]
In the last attempt, I registered an account and logged in, and then suddenly a "setup-code.txt" file appeared in the folder. I found an 8-digit number and used /setup code when I entered the server. When I was told "your new account has been verified, and the /setup system has been turned off.",
If you log out and use again, you will be told "
the initial setup system is disabled. This incident has been logged. If you are locked out of all admin accounts, ask for help on https://tshock.co/".

**Reply by hakusaro** [reply]
It sounds like you set it up correctly though?

**Reply by viewfirst** [reply]
In the end, I didn't find a way to obtain the permissions, and now I am going to restart the server. thanks for your help

**Comment by hakusaro** [comment]
Which container are you using?

On October 24, 2020, GitHub <notifications@github.com> wrote:
> For some reason, I have deleted the docker container and selected an

> existing world when creating a new server. The file you mentioned is

> not in the "world" folder. Is there any other way to help me?
>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Pryaxis/TShock/discussions/2141#discussioncomment-

110643>, or unsubscribe <https://github.com/notifications/unsubscribe-

auth/AAECBJ3CHTD26P4UGAIEKK3SMMFGJANCNFSM4S5W273Q>.

**Reply by hakusaro** [reply]
man, this sucks

**Reply by viewfirst** [reply]
https://hub.docker.com/r/ryshe/terraria


In the last attempt, I registered an account and logged in, and then suddenly a "setup-code.txt" file appeared in the folder. I found an 8-digit number and used /setup code when I entered the server. When I was told "your new account has been verified, and the /setup system has been turned off.",
If you log out and use again, you will be told "
the initial setup system is disabled. This incident has been logged. If you are locked out of all admin accounts, ask for help on https://tshock.co/".
